library/button/button.css

```diff
@@ -0,0 +1,109 @@
+.btn {
+  padding: 0.5rem;
+  margin: 0.5rem;
+  border: 0;
+  border-radius: 5px;
+  font-weight: 600;
+}
+
+.btn:hover {
+  box-shadow: 1px 1px 2px rgba(128, 128, 128, 0.986);
+  cursor: pointer;
+}
+
+.btn-primary {
+  background-color: var(--primary-color);
+}
+
+.btn-primary:hover {
+  background-color: rgba(252, 127, 25, 0.631);
+}
+
+.btn-success {
+  background-color: rgb(0, 156, 0);
+}
+
+.btn-success:hover {
+  background-color: rgba(0, 156, 0, 0.5);
+}
+
+.btn-danger {
+  background-color: rgb(255, 37, 37);
+}
+
+.btn-danger:hover {
+  background-color: rgba(255, 37, 37, 0.5);
+}
+
+.btn-disabled {
+  background-color: rgba(128, 128, 128, 0.6);
+}
+
+.btn-disabled:hover {
+  cursor: not-allowed;
+  background-color: rgba(128, 128, 128, 0.3);
+}
+
+.btn-outline {
+  background-color: rgb(255, 255, 255);
+}
+
+.btn-outline.btn-primary {
+  border: 1px solid var(--primary-color);
+  color: var(--primary-color);
+}
+
+.btn-outline.btn-primary:hover,
+.btn-outline.btn-success:hover,
+.btn-outline.btn-danger:hover,
+.btn-outline.btn-disabled:hover {
+  color: white;
+  font-weight: 700;
+}
+.btn-outline.btn-success {
+  border: 1px solid rgb(0, 112, 0);
+  color: rgb(0, 112, 0);
+}
+
+.btn-outline.btn-danger {
+  border: 1px solid rgb(207, 19, 19);
+  color: rgb(207, 19, 19);
+}
```

Comment:
Try to use hex values for colors than RGB. Other than that all good

Reply:
Yes will do this in next iteration